src/main/java/de/zalando/ep/zalenium/registry/ZaleniumRegistry.java

```diff
@@ -58,6 +62,23 @@
     private final Matcher matcherThread = new Matcher();
     private final List<RemoteProxy> registeringProxies = new CopyOnWriteArrayList<>();
     private volatile boolean stop = false;
+    
+    private static final Environment defaultEnvironment = new Environment();
+    
+    private static final double[] DEFAULT_TEST_SESSION_LATENCY_BUCKETS =
+            new double[] { 0.5,2.5,5,10,15,20,25,30,35,40,50,60 };
+
+    // Allows overriding of the test session latency buckets for prometheus
+    private static final String ZALENIUM_TEST_SESSION_LATENCY_BUCKETS = "ZALENIUM_TEST_SESSION_LATENCY_BUCKETS";
```

Comment:
Here. I didn’t bother to plumb it through zalenium.sh

Reply:
Adding it to zalenium.sh is only needed if you want to use command line args. Can add that if you like. But that’s more of a docker thing. 

Maybe we should have a table in the documentation that shows mapping’s between environment variables and command line args where they exist.

Reply:
OK, so I think we were talking about different variables. Since in the initial comment you mention [`NEW_SESSION_WAIT_TIMEOUT`](https://github.com/zalando/zalenium/blob/9abf3a2d26d373e4be6611802cc6dad01489ce5d/scripts/zalenium.sh#L25), I thought that one was changing. 

But I guess you were only talking about `ZALENIUM_TEST_SESSION_LATENCY_BUCKETS`, all good :)

Reply:
Oh yeah, sorry. I mentioned the `NEW_SESSION_WAIT_TIMEOUT` as that is how I tested test session requests being timed out, and the metrics still decrementing properly.